instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/engine.py

```diff
@@ -48,27 +48,62 @@ def _get_tracer(tracer_provider=None):
     )
 
 
-def _wrap_create_async_engine(tracer_provider=None):
+class EngineTracerOptions:
+    """Configuration for the EngineTracer.
+
+    Args:
+        ``enable_commenter``: enables the sql commenter
+        ``commenter_options``: options for the sql commenter
+        ``before_execute_hook``: a callback for the cursor execute with span and query context.
```

Comment:
convention is to use `XXX_callback` -> `execute_callback` would make more sense.

Reply:
Might as well add `after_cursor_execute_callback` and `handle_error_callback`